### ISSUE TITLE
Create two PDFs - one sided (for eReaders) and two-sided (for printing)

### ISSUE THREAD

**Issue by hmemcpy** [hmemcpy/milewski-ctfp-pdf#100]
Some hints on how to do this in latex:

https://tex.stackexchange.com/questions/300252/generate-oneside-and-twoside-version-at-the-same-time
https://tex.stackexchange.com/questions/220091/conditional-compilation-of-documents-using-command-line-options-for-pdflatex

**Comment by mseri** [comment]
I think it would make sense to split the preamble and take out the geometry information, than we could have different pre-preambles with the right margins and font sizes for print and epub. It could also include a `pdfbook` parsed version for people that want to print it in booklet form.

**Comment by hmemcpy** [comment]
Yeah, the preamble is way too big as it is... I trimmed it down, but there are still things that make the PDF beautiful :)
It is based on [this SICP PDF](https://github.com/sarabander/sicp-pdf)'s preamble. Any refactoring work there would be welcomed!
